pytest_cases/tests/simple/test_paramfixtures.py

```diff
@@ -13,38 +13,43 @@ def pytest_param(*args, **kwargs):
 
 
 # create a single parameter fixture
-my_parameter = param_fixture("my_parameter", [1, 2, 3, 4])
+param_fixture("my_parameter", [1, 2])
+my_parameter2 = param_fixture("my_parameter2", [3, 4])  # Returning value
 
 
 @pytest.fixture
-def fixture_uses_param(my_parameter):
-    return my_parameter
+def fixture_uses_param(my_parameter, my_parameter2):
+    return my_parameter, my_parameter2
 
 
-def test_uses_param(my_parameter, fixture_uses_param):
+def test_uses_param(my_parameter, my_parameter2, fixture_uses_param):
     # check that the parameter injected in both is the same
-    assert my_parameter == fixture_uses_param
+    assert my_parameter, my_parameter2 == fixture_uses_param
 
 
 # -----
 # create a 2-tuple parameter fixture
 arg1, arg2 = param_fixtures("arg1, arg2", [(1, 2), (3, 4)])
+# Testing fix injection and single arg
+# Renaming to "parg3" solves the problem
+param_fixtures("arg3", [5, 6])
```

Comment:
This fixture interferes with the parameter `arg3` in the parametrization at line 70.

Reply:
I think that this is normal pytest behaviour. If you create a fixture and then create a parameter with the same name there will be a conflict.

Reply:
Well I tried the following without a problem, it uses the "fixture" with the most inner scope (in this case, the one from parametrized:

```
import pytest

@pytest.fixture
def arg():
    return "arg"


@pytest.mark.parametrize("arg", [1, 2])
def test_one(arg):
    print(arg)
```

result:

```
(venv) D:\testing-pytest-fixture-and-param-behaviour>pytest -sv
=============================================================================================== test session starts ================================================================================================
platform win32 -- Python 3.6.8, pytest-4.4.0, py-1.8.0, pluggy-0.9.0 -- d:\testing-pytest-fixture-and-param-behaviour\venv\scripts\python.exe
cachedir: .pytest_cache
rootdir: D:\testing-pytest-fixture-and-param-behaviour
collected 2 items                                                                                                                                                                                                   

test_one.py::test_one[1] 1
PASSED
test_one.py::test_one[2] 2
PASSED

============================================================================================= 2 passed in 0.02 seconds =============================================================================================

```

Reply:
ok, I'll have a look then. I will open a dedicated issue